.circleci/config.yml

```diff
@@ -138,38 +155,49 @@ jobs:
                --label "build.number=$CIRCLE_BUILD_NUM" \
                --label "build.url=$CIRCLE_BUILD_URL" \
                --label "build.gitref=$CIRCLE_SHA1"
+      - snyk/scan:
+          project: '${CIRCLE_PROJECT_REPONAME}-docker/${CIRCLE_BRANCH}'
+          docker-image-name: 'mojdigitalstudio/court-case-service:$APP_VERSION'
+          target-file: 'Dockerfile'
+          monitor-on-build: << parameters.master >>
+          <<: *snyk_options
+      - when:
+          condition: << parameters.master >>
+          steps:
+            - run:
+                name: Publish to docker hub
+                command: |
+                  docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+                  docker tag mojdigitalstudio/court-case-service:$APP_VERSION mojdigitalstudio/court-case-service:latest
+                  docker push mojdigitalstudio/court-case-service:$APP_VERSION
+                  docker push mojdigitalstudio/court-case-service:latest
+                  fi
 
-            if [[ "${CIRCLE_BRANCH}" == "master" ]]; then
-              # publish to docker hub as the tag
-             docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-             docker tag mojdigitalstudio/court-case-service:$APP_VERSION mojdigitalstudio/court-case-service:latest
-             docker push mojdigitalstudio/court-case-service:$APP_VERSION
-             docker push mojdigitalstudio/court-case-service:latest
-            fi
-
-  app-scan:
+  app_scan:
     executor: builder
+    parameters:
+      master:
+        type: boolean
+        default: false
     steps:
       - checkout
       - snyk/scan:
-          fail-on-issues: false
-          monitor-on-build: true
           project: '${CIRCLE_PROJECT_REPONAME}/${CIRCLE_BRANCH}'
-          organization: 'digital-probation-services'
+          monitor-on-build: << parameters.master >>
+          <<: *snyk_options
 
-  image-scan:
+  image_scan:
```

Comment:
why underscore - next job uses hyphen ?

Reply:
i was trying to improve consistency with job naming e.g. `build_docker` `deploy_whatever`. but i can see there are jobs with dashes too `request-preprod-approval`, `sonar-scan`.

Reply:
I agree - we need to get some consistency

Reply:
changed the other jobs to use underscores too